### PR TITLE
EZP-24367: Ensures RestAuthenticator is not rememberme aware

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-24367

Complements https://github.com/ezsystems/LegacyBridge/pull/26

### CODE REVIEW

**Comment by lolautruche** [comment]
Review ping @bdunogier @andrerom @dpobel @yannickroger @pspanja

**Comment by bdunogier** [comment]
1

File: eZ/Bundle/EzPublishRestBundle/DependencyInjection/Security/RestSessionBasedFactory.php [hunk 1]
```diff
@@ -23,6 +23,11 @@ public function __construct()
         $this->defaultFailureHandlerOptions = array();
     }
 
+    protected function isRememberMeAware($config)
```
Comment (blankse) [inline eZ/Bundle/EzPublishRestBundle/DependencyInjection/Security/RestSessionBasedFactory.php]: CS?

Comment (lolautruche) [inline eZ/Bundle/EzPublishRestBundle/DependencyInjection/Security/RestSessionBasedFactory.php]: Nice catch @blankse, thanks :smile: . Struggling a bit between PSR-2 and eZ CS

**Comment by pspanja** [comment]
+1

File: eZ/Bundle/EzPublishRestBundle/DependencyInjection/Security/RestSessionBasedFactory.php (hunk 1 above)
Comment (blankse) [inline eZ/Bundle/EzPublishRestBundle/DependencyInjection/Security/RestSessionBasedFactory.php]: Why eZ Robot doesn't catch this?

**Pull request merged**